packages/popover/src/index.tsx

```diff
@@ -144,12 +146,17 @@ const usePopover = ({
       ...getPlacement(targetRect, popoverRect)
     };
 
+    if (isFlipped !== shouldFlip) {
+      setIsFlipped(shouldFlip);
+    }
+
     setStyles((prev) => ({
       ...prev,
       ...newPosition
     }));
   }, [
     hidden,
+    children,
```

Comment:
please update the storybook with examples that popover respects the children's changes. for example, height.
even if the popover is flipped or not.

Reply:
I added the example but I have a problem when trying to start storybook. I tried with develop and main branches wo/any changing, but still the same error. 
```
ERROR in ./packages/accordion/src/index.tsx
Module build failed (from ./node_modules/ts-loader/index.js):
TypeError: Cannot read property 'tap' of undefined
    at makeAssetsCallback (/Users/oguzhan.sofuoglu/ciceksepeti/cactus-ui-1/node_modules/ts-loader/dist/instances.js:218:50)
    at addAssetHooks (/Users/oguzhan.sofuoglu/ciceksepeti/cactus-ui-1/node_modules/ts-loader/dist/instances.js:224:9)
    at Object.initializeInstance (/Users/oguzhan.sofuoglu/ciceksepeti/cactus-ui-1/node_modules/ts-loader/dist/instances.js:288:9)
    at successLoader (/Users/oguzhan.sofuoglu/ciceksepeti/cactus-ui-1/node_modules/ts-loader/dist/index.js:26:17)
    at Object.loader (/Users/oguzhan.sofuoglu/ciceksepeti/cactus-ui-1/node_modules/ts-loader/dist/index.js:23:5)
 @ ./packages/accordion/src/stories/accordion.stories.mdx 14:8-21
 @ ./packages sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(tsx|mdx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-server/node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-server/node_modules/@storybook/core-client/dist/esm/globals/globals.js ./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/addon-docs/dist/esm/frameworks/common/config.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/dist/esm/frameworks/react/config.js-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-a11y/dist/esm/a11yRunner.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/dist/esm/a11yHighlight.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/dist/esm/preset/addDecorator.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/dist/esm/preset/addArgs.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/dist/esm/preset/addDecorator.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/dist/esm/preset/addParameter.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/dist/esm/preset/addDecorator.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/dist/esm/preset/addDecorator.js-generated-config-entry.js ./generated-stories-entry.js
```

Sometimes build succeeded (i think it's completely random. once the build succeeded then I tried again the same command and the build failed). I've researched this error and as far as I understand it points webpack issue, maybe related webpack version?